ci-operator/config/openshift/cluster-dns-operator/master.json

```diff
@@ -25,7 +25,7 @@
     {
       "from": "base",
       "to": "cluster-dns-operator",
-      "dockerfile_path": "Dockerfile.cluster-dns-operator"
+      "dockerfile_path": "Dockerfile"
```

Comment:
You don't need to specify this, it is the default

Reply:
Removed it from both operator definitions, thanks